scripts/ManageAutoroutes.user.js

```diff
@@ -15,42 +15,47 @@
+    //if you dont own the hub yet, the button will be disabled in that galaxy, but will become active when you colonize that hub.
     var hubGalaxy1 = "ishtar gate";
     var hubGalaxy2 = "solidad";
     var hubGalaxy3 = "xirandrus";
     var shipsPerAutoroute = 1;
-    var transferPercent = 101;
-    var useLategameExclusions = false; //true => no shipping of plastic except for radioactive/acid and no shipping of graphite/titanium for lava *** false => transfer all resources
-    var noDuplicates = true; // true => do not create autoroute if target planet already has one
-    var takeFromHubfleet = true; //true => try to split ships from hubfleet (buy ships only if there aren't enough) **** false => always buy ships
-    var topUp = true; // true => if there is already an autoroute, will try to build enough ships to stop any present overloading (will also set noDuplicates to true).
-    var redundancy = 10; //percent of extra ships to build, useful to "make space" for future increases in production. Set to zero to disable. (won't affect the "shipsPerAutoroute" value).
-    var cancelOldroutes = true; //true => if using andromeda and the current route contains orions, split them out + cancel travel *** false => normal behaviour (but issue a warning in the console)
+    var transferPercent = 101;  // The tansfer rate to check on non-hub side of the auto-route -- in-game defaule is 105
+    var useLategameExclusions = false; // false => transfer all resources *** true => exclude plastic except for radioactive/acid planets and exclude graphite/titanium on lava planets
+    var noDuplicates = true; // true => do not create autoroute if target planet already has one *** false => create a new autoroute every time -- NOT RECOMMENDED
+    var takeFromHubFleet = true; //true => try to split ships from hubfleet (buy ships only if there aren't enough) **** false => always buy ships
+    var topUp = true; // true => if there is already an autoroute, will try to build enough ships to stop any present overloading (will also set noDuplicates to true). false => script will not interact with routes that already exist.
+    var redundancy = 10; //percent of extra ships to build, useful to "make space" for future increases in production. Set to zero to disable. (won't affect the "shipsPerAutoroute" value).  MUST BE 0 OR POSITIVE
+    var cancelOldRoutes = true; //true => if using andromeda and the current route contains orions, split them out + cancel travel *** false => normal behaviour (but issue a warning in the console)
+
+	// When the script (auto)updates this will be set to false various acitons can cause stored values to be deleted, so it is recommended that you restore the above values to your liking and set this back to true
+    var ignoreStoredValues = false; //true => save the above options on next reload of game page. false => load the last saved options. 
```

Comment:
I think these descriptions are much clearer, and better after going through the other variables.